src/commands/deploy/buildImageInAzure/RootFolderStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { AzureWizardPromptStep, UserCancelledError } from '@microsoft/vscode-azext-utils';
+import * as vscode from 'vscode';
+import { localize } from '../../../utils/localize';
+import { IBuildImageInAzureContext } from './IBuildImageInAzureContext';
+
+export class RootFolderStep extends AzureWizardPromptStep<IBuildImageInAzureContext> {
+    public async prompt(context: IBuildImageInAzureContext): Promise<void> {
+        context.rootFolder = await getRootWorkSpaceFolder();
+    }
+
+    public shouldPrompt(context: IBuildImageInAzureContext): boolean {
+        return !context.rootFolder;
+    }
+}
+
+export async function getRootWorkSpaceFolder(): Promise<vscode.WorkspaceFolder> {
```

Comment:
Do you need to export this? I don't see it being imported anywhere.

If it is, I think we should probably move it to the `workspaceUtils`.

Reply:
Not using it anywhere else 😄 I'll remove the export.